src/app/[locale]/post-security/sidebar.tsx

```diff
@@ -36,8 +36,13 @@ export function PostSecuritySidebar({
   const [hash, setHash] = useState<string>("");
   const [isClearing, setIsClearing] = useState(false);
   const { setOpenMobile } = useSidebar();
+  const locale = useLocale();
   const t = useTranslations("post-security-page");
 
+  // RTL languages
+  const isRTL = locale === "ar";
```

Comment:
This needs to be refactored somewhere, the comparison is done in multiple files and doesn't adhere to DRY.